teuthology/config.py

```diff
@@ -186,6 +186,10 @@ class TeuthologyConfig(YamlConfig):
             },
         },
         'sleep_before_teardown': 0,
+        'container_images': {
+            'base_registry_mirror': 'vossi04.front.sepia.ceph.com:5000',
+            'ci_image': 'quay.io/ceph-ci/ceph'
+        }
```

Comment:
As we figured out earlier this is test specific config, it should not be a part of teuthology code base.

Reply:
another way to look these settings is see it as a freestyle setting which can be consumed by any test(s). we could put these settings in their own namespace though, if that's better.